dds_cli/custom_decorators.py

```diff
@@ -52,7 +52,7 @@ def wrapped(self, file, *args, **kwargs):
 
         # Mark as started
         self.status[file]["started"] = True
-        LOG.debug("File '%s' started: %s", escape(str(file)), func.__name__)
+        LOG.debug("%s: started file '%s'", func.__name__, escape(str(pathlib.Path(file).name)))
```

Comment:
Why do we need to add the pathlib.Path here? I feel like we'll lose some info. Sometimes there are multiple files with the same name but in different directories, so it could become difficult figuring out the exact file if one of them is having issues.